lib/bambora/json_client.rb

```diff
@@ -89,38 +93,15 @@ def post(path:, body:, api_key:)
     #
     # @return [Hash] Indicating success or failure of the operation.
     def delete(path:, api_key:)
-      parse_response(
-        super(path: path, headers: build_headers(api_key)),
-      )
+      parse_response_body(
+        super(path: path, headers: build_headers(api_key: api_key)),
+      ).to_h
```

Comment:
I think the `#to_h` is unnecessary here, but I don't think `{}.to_h.to_h` will cause an exception or return incorrect data.

Reply:
Right right. I'm going to merge this and add a follow up to get rid of the extra `to_h`.